### PR TITLE
add roadmap

### PR DESCRIPTION
**Pull request by AdrianRossouw**
this is the beginning of a roadmap for future releases.

needs more explanatory language

### CODE REVIEW

**Comment by AdrianRossouw** [comment]
@rjrodger @pelger @mcdonnelldean @naomifeehanmoran 

you guys have more insight into this, please check the language and help us narrow it down a bit.

File: src/pages/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+template: main.html
+title: Seneca Roadmap
+---
+
+## Seneca Roadmap
+
+Seneca tries to follow a quartly release schedule with major changes gated behind subsequented releases.
+This is to enforce the continuity of development, and to ensure that semver is properly followed.
+These goals and may change at any time based on challenges encountered or contributions from the community.
```
Comment (naomifeehanmoran) [inline src/pages/roadmap.md]: possible typo: "These **goals and may** change"

**Comment by naomifeehanmoran** [comment]
thanks @AdrianRossouw for getting this started 👍

File: src/pages/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### 3.0 - June 2016
+* Transport refactor
+* Replace logging with Pino
+* Error handling refactor
+* Deprecate seneca.pin
+* Remove seneca web core dependency
+
+### 4.0 - August 2016
+* Plugins?
```
Comment (mcdonnelldean) [inline src/pages/roadmap.md]: Lets remove the question marks

**Comment by mcdonnelldean** [comment]
@AdrianRossouw couple of edit suggestions from myself and @naomifeehanmoran. Other than that, looks good. We can improve as we get more context and feedback.

**Pull request merged**